TORCH_XLA_USER_GUIDE.md

```diff
@@ -97,9 +97,13 @@ PJRT_DEVICE=TPU XLA_FLAGS=--xla_dump_to=/tmp/dir_name PROFILE_LOGDIR=/tmp/home/
 that you have XLA:GPU support. Please refer to [pytorch/xla](https://github.com/pytorch/xla#wheel) repo to download
 a suitable GPU nightly wheel for your environment.
 
-After that, you can run the following the command:
+Then, to run with XLA:GPU:
 ```
-PJRT_DEVICE=GPU GPU_NUM_DEVICES=8 python3 example_text_completion.py 1 --ckpt_dir . --tokenizer_path $HOME/llama/t5_tokenizer/spiece.model --max_seq_len 2048 --max_gen_len 1000 --max_batch_size 1 --dynamo True --mp True
+PJRT_DEVICE=CUDA GPU_NUM_DEVICES=8 python3 example_text_completion.py 1 --ckpt_dir . --tokenizer_path $HOME/llama/t5_tokenizer/spiece.model --max_seq_len 2048 --max_gen_len 1000 --max_batch_size 1 --dynamo openxla --mp True
+```
+You can also run the same with Inductor:
+```
+CUDA_VISIBLE_DEVICES=0 USE_CUDA=1 python3 example_text_completion.py 1 --ckpt_dir . --tokenizer_path $HOME/llama/t5_tokenizer/spiece.model --max_seq_len 2048 --max_gen_len 1000 --max_batch_size 1 --dynamo inductor --mp True
```

Comment:
Also need to update the other `--dynamo True/False` usage for TPU in this guide?

Reply:
Good catch! Done.
Another small change I made is to pass openxla_eval to preserve old behavior.